js_modules/dagster-ui/packages/ui-core/src/assets/AssetNodeOverview.tsx

```diff
@@ -81,34 +84,50 @@ import {buildRepoAddress} from '../workspace/buildRepoAddress';
 import {workspacePathFromAddress} from '../workspace/workspacePath';
 
 export const AssetNodeOverview = ({
+  assetKey,
   assetNode,
   upstream,
   downstream,
   liveData,
   dependsOnSelf,
 }: {
-  assetNode: AssetNodeDefinitionFragment;
+  assetKey: AssetKey;
+  assetNode: AssetNodeDefinitionFragment | undefined | null;
   upstream: AssetNodeForGraphQueryFragment[] | null;
   downstream: AssetNodeForGraphQueryFragment[] | null;
   liveData: LiveDataForNode | undefined;
   dependsOnSelf: boolean;
 }) => {
-  const repoAddress = buildRepoAddress(
-    assetNode.repository.name,
-    assetNode.repository.location.name,
-  );
+  const repoAddress = assetNode
+    ? buildRepoAddress(assetNode.repository.name, assetNode.repository.location.name)
+    : buildRepoAddress('', '');
```

Comment:
nit: this invalid repo address feels a bit goofy, maybe worth using `null` and updating things to allow that ?

Reply:
Let me try, I was worried it might blow up but maybe it won't 😬